lib/pf/web/dispatcher.pm

```diff
@@ -53,205 +52,186 @@ Reference: http://perl.apache.org/docs/2.0/user/handlers/http.html#PerlTransHand
 sub handler {
     my $r = Apache::SSLLookup->new(shift);
     my $logger = Log::Log4perl->get_logger(__PACKAGE__);
-    $logger->trace("hitting translator with URL: " . $r->uri);
-    # Test if the hostname is include in the proxy_passthroughs configuration
-    # In this case forward to mad_proxy
-    if ( ( ($r->hostname.$r->uri) =~ /$PROXYPASSTHROUGH::ALLOWED_PASSTHROUGH_DOMAINS/o && $PROXYPASSTHROUGH::ALLOWED_PASSTHROUGH_DOMAINS ne '') || ($r->hostname =~ /$PROXYPASSTHROUGH::ALLOWED_PASSTHROUGH_REMEDIATION_DOMAINS/o && $PROXYPASSTHROUGH::ALLOWED_PASSTHROUGH_REMEDIATION_DOMAINS ne '') ) {
-        my $parsed_request = APR::URI->parse($r->pool, $r->uri);
-        $parsed_request->hostname($r->hostname);
-        $parsed_request->scheme('http');
-        $parsed_request->scheme('https') if $r->is_https;
-        $parsed_request->path($r->uri);
-        return proxy_redirect($r, $parsed_request->unparse);
-    }
 
-    #Apache Filtering
+    my $hostname = $r->hostname;
+    my $uri = $r->uri;
+    my $url = $r->construct_url;
+    my $user_agent = $r->headers_in->{'User-Agent'};
+
+    $logger->debug("hitting handler with URI '$uri' (URL: $url)");
```

Comment:
I think that it's a bit too verbose as we sometime leave logs in debug in production to capture data.

This is going to destroy @lzammit eyes